src/unity/python/turicreate/toolkits/recommender/util.py

```diff
@@ -499,7 +501,8 @@ def random_split_by_user(dataset,
             'item_test_proportion': item_test_proportion,
             'random_seed': random_seed}
 
-    response = _turicreate.extensions._recsys.train_test_split(opts)
+    response = _turicreate.extensions._recsys.train_test_split(dataset, user_id, item_id, max_num_users, item_test_proportion, random_seed)
```

Comment:
This line is a bit long. I try to limit my line length to under a 100 characters. I think the official guidance is under 80. I'd recommend breaking this up into more than one line.

Reply:
Thanks for pointing it out!